main/led.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#define LED_RED_PIN PA2;
```

Comment:
```suggestion
#define LED_RED_PIN PA2
```
remember that everything starting with `#` is a macro and thus not C/C++.
Your current statement replaces `LED_RED_PIN` with `PA2;` which throws compile errors because you paste random semicola into your code

Reply:
This is weird. For some reason it compiles for me... but should be fixed now